src/menu/styled-components.js

```diff
@@ -27,6 +27,10 @@ export const StyledList = styled<StyledPropsT>('ul', ({$theme}) => {
     paddingBottom: $theme.sizing.scale300,
     paddingLeft: 0,
     paddingRight: 0,
+    borderTopLeftRadius: $theme.borders.popoverBorderRadius,
+    borderTopRightRadius: $theme.borders.popoverBorderRadius,
+    borderBottomRightRadius: $theme.borders.popoverBorderRadius,
+    borderBottomLeftRadius: $theme.borders.popoverBorderRadius,
```

Comment:
Feels strange to tie the menu to popover props since it's only a single use case of the menu component.

Reply:
Yeah, I did so because pre `v7` they both had border radius `8px` and it kind of made sense to me that menu was an instance of a popover. Are there cases where it is not a popover? If so I may need to make a new variable for it.